host/examples/parsec_execute/src/main.rs

```diff
@@ -0,0 +1,105 @@
+//! An example to send a file.
+
+use anyhow::anyhow;
+use log::{error, info};
+use utils::attestation;
+use transport::{messages::{Request, Response, Status}, session::Session};
+
+use std::fs;
+use std::fs::File;
+use std::io::Read;
+
+fn get_file_as_byte_vec(filename: &String) -> Vec<u8> {
```

Comment:
It is better practice to return an error than panicking. I recommend the use of anyhow errors (cf. main())